scripts/superdesk/directives/sdPasswordStrength.js

```diff
@@ -0,0 +1,111 @@
+(function() {
+    'use strict';
+
+    // config holds the configuration for the password strength calculator.
+    var config = {
+        MIN_LENGTH: 8,
+        ONE_LOWER: /^(?=.*[a-z])/g,
+        ONE_UPPER: /^(?=.*[A-Z])/g,
+        ONE_NUMBER: /^(?=.*[0-9])/g,
+        ONE_OTHER: /^(?=.*[^0-9^a-z^A-Z])/g,
+        MIN_STRENGTH: 3
+    };
+
+    // styles holds each of the strength labels by index along with the class
+    // to be added to the indicator.
+    var styles = [
+        {txt: 'Short', cls: 'red'},
```

Comment:
here it should use `gettext` for those txt values - it's used later so it would translate it, but without it being here it won't be picked when generating strings to translate